src/Statement/MysqlStatementBuilder.php

```diff
@@ -4,13 +4,32 @@
 
 use Exo\Operation\AbstractOperation;
 use Exo\Operation\ColumnOperation;
+use Exo\Operation\FunctionOperation;
 use Exo\Operation\IndexOperation;
+use Exo\Operation\ParameterOperation;
 use Exo\Operation\TableOperation;
 use Exo\Operation\UnsupportedOperationException;
+use Exo\Operation\VariableOperation;
 use Exo\Operation\ViewOperation;
 
 class MysqlStatementBuilder extends StatementBuilder
 {
+    const VIEW_CREATE_OR_REPLACE = 'CREATE OR REPLACE VIEW %s AS (%s);';
+    const VIEW_DROP = 'DROP VIEW %s;';
+    const FUNCTION_CREATE_OR_REPLACE = '
```

Comment:
Is there a need for idempotency when creating functions? I would expect this operation to fail if the functional already exists, just as I would expect the operation to fail when I try to create an existing table.

Reply:
I don't know that there is an explicit need for it, it just seemed the simpler solution for items that are self encapsulated. I am open to change this, but considering the entire state of the view and or function is encapsulated in the definition, I saw no risk as well as convenience & simplicity. If you're ok with it, I'd like to go with this for now and swing back to update this in the near term.

Reply:
For consistency I think it's best that create and alter result in different SQL being issued, such that create fails if the function/view already exists.

Reply:
Done.